tasks/install.yml

```diff
@@ -71,9 +79,3 @@
     state: link
   become: true
 
-- name: Add pantheon to system-wide $PATH
-  copy:
-    dest: "{{ pantheon_profile_file }}"
-    content: 'PATH=$PATH:{{ pantheon_current_dir }}/bin'
-  become: true
-  when: ansible_os_family != "Darwin"
```

Comment:
Did we mean to lose this and not have pantheon on the path?

Reply:
Yup, its unused so figured I'll clean it up